microbench/src/main/java/io/netty/microbench/concurrent/BurstCostExecutorsBenchmark.java

```diff
@@ -297,26 +183,19 @@ public int spinWaitCompletionOf(int value) {
     }
 
     @Benchmark
-    @BenchmarkMode(Mode.SampleTime)
+    @BenchmarkMode(Mode.AverageTime)
     @Threads(1)
     public int test1Producer(final PerThreadState state) {
         return executeBurst(state);
     }
 
     @Benchmark
-    @BenchmarkMode(Mode.SampleTime)
+    @BenchmarkMode(Mode.AverageTime)
     @Threads(2)
     public int test2Producers(final PerThreadState state) {
         return executeBurst(state);
     }
 
-    @Benchmark
-    @BenchmarkMode(Mode.SampleTime)
-    @Threads(3)
-    public int test3Producers(final PerThreadState state) {
```

Comment:
Just matter of math on common machines: thoeretically is not a good idea to run benchs on a dev machine, but sometime it could happen; many dev machines has just 4 physical cores ie a run with 2 consumers and 2 producers is already maximizing the machine. Using 5 cores (3 producers + 2 consumers) could just produce werid results...